tests/acceptance/test_mender-connect.py

```diff
@@ -286,10 +286,7 @@ def test_mender_connect_reconnect(
 
             # wait for error
             _ = wait_for_string_in_log(
-                connection,
-                startup_time,
-                300,
-                "eventLoop: error reconnecting: failed to connect after max number of retries",
+                connection, startup_time, 300, "connection manager failed to connect",
```

Comment:
This repository is versioned on a different schedule than client components, meaning that  a give version of `meta-mender` needs to be able to build and test several versions of `mender-connect` for example. So we need modify the test to be able to verify either version.

It is the first time for you encountering this problem, but it could be something like this:
```
            # wait for error
            if version_is_minimum(bitbake_variables, "mender-connect", "2.3.0"):
                error_message = "connection manager failed to connect"
            else:
                error_message = "eventLoop: error reconnecting: failed to connect after max number of retries"
            _ = wait_for_string_in_log(connection, startup_time, 300, error_message)
```

Plus adding `bitbake_variables` to the fixtures to be used in this test and importing `version_is_minimum`.